baseplate/experiments/__init__.py

```diff
@@ -189,8 +201,36 @@ def variant(self, name, user=None, bucketing_event_override=None,
 
         return variant
 
+    def expose(self, experiment_name, variant_name, user=None, **kwargs):
+
+        experiment = self._get_experiment(experiment_name)
+
+        if experiment is None:
+            return
+
+        inputs = dict(kwargs)
+
+        if user:
+            if user.is_logged_in:
+                inputs["user_id"] = user.id
+                inputs["user_roles"] = user.roles
+            else:
+                inputs["user_id"] = user.loid
+            inputs["logged_in"] = user.is_logged_in
+            inputs['cookie_created_timestamp'] = user.event_fields().get('cookie_created')
+
+        self._event_logger.log(
+            experiment=experiment,
+            variant=variant_name,
+            user_id=inputs.get('user_id'),
+            logged_in=inputs.get('logged_in'),
+            cookie_created_timestamp=inputs.get('cookie_created_timestamp'),
+            app_name=inputs.get('app_name'),
+            exposed=True,
+        )
+
 
-def experiments_client_from_config(app_config, event_queue):
+def experiments_client_from_config(app_config, event_logger):
```

Comment:
Is anything currently using this?  This update isn't backward-compatible so it could cause issues if something is using this already.

Reply:
The only caller of this right now within the organization is messenger service, and it's using postitional parameters and sending in an EventLogger. This is not a functional change from the current state. The ExperimentsContextFactory already expects an EventLogger, not an EventQueue. It's just a param rename.

Though I did miss renaming it in the relevant test. Will update that.